lib/Reference/ContentReferenceHelper.php

```diff
@@ -108,6 +108,8 @@ public function resolveReference(string $referenceText): ?IReference {
 
 			$referenceInfo['ownership'] = $element->getOwnership();
 			$referenceInfo['ownerDisplayName'] = $element->getOwnerDisplayName();
+			$referenceInfo['isShared'] = $element->getIsShared();
+			$referenceInfo['onSharePermissions'] = $element->getOnSharePermissions();
```

Comment:
Couldn't see this being used, is it somewhere?

Reply:
> Couldn't see this being used, is it somewhere?

Yeah, so the `NcTable` component passes some permissions like this to child components, and it turns out in the widget this information isn't actually there, so I added those to the response from the backend so we can pass it to the children from the widget component. Maybe there's a better way?

Reply:
No sounds good then